src/shapes/circle.py

```diff
@@ -0,0 +1,28 @@
+import cv2
+from numpy import ndarray
+
+from src.shapes.point import Point
+from src.shapes.shape import Shape
+
+
+class Circle(Shape):
+    def __init__(self, radius, center: Point, line_color: tuple[int, int, int], image: ndarray):
+        self.center = center
+        self.radius = radius
+        self.line_color = line_color
+        self.image = image
+
+    def draw(self) -> None:
```

Comment:
I think that instead of each shape holding its own version of the image, it is better for draw to receive image at the moment the drawing is requested